Wrapping/Generators/Python/itk/support/helpers.py

```diff
@@ -173,3 +176,123 @@ def image_filter_wrapper(*args, **kwargs):
             return image_filter(*args, **kwargs)
 
     return image_filter_wrapper
+
+
+def wasm_type_from_image_type(itkimage):  # noqa: C901
+    import itk
+
+    component = itk.template(itkimage)[1][0]
+    if component == itk.UL:
+        if os.name == 'nt':
+            return 'uint32_t', 1
+        else:
+            return 'uint64_t', 1
+    mangle = None
+    pixelType = 1
+    if component == itk.SL:
+        if os.name == 'nt':
+            return 'int32_t', 1,
+        else:
+            return 'int64_t', 1,
+    if component in (itk.SC, itk.UC, itk.SS, itk.US, itk.SI, itk.UI, itk.F,
```

Comment:
Are `long` and `long long` types absent from this list on purpose?

Reply:
It looks like this list of types was first introduced by @thewtex in this commit [ENH: Use strong component typing _image_to_type](https://github.com/InsightSoftwareConsortium/itkwidgets/commit/88c0e659eca065c46a0eb5e4700d481dca398f78), which was part of this PR [InsightSoftwareConsortium/itkwidgets #323 Unsigned long pixel type](https://github.com/InsightSoftwareConsortium/itkwidgets/pull/323). 

@thewtex can you tell us if we should be adding `long` and `long long` types in this context?

Reply:
Yes, can we please add `long` and `long long` type support by adding `itk.L`, `itk.LL`, `itk.UL`, `itk.ULL` to this list? These were originally not added because JavaScript did not have support, but that is historical now.

Reply:
Pretty sure that it needs to be `itk.SL`, `itk.SLL`, `itk.UL`, `itk.ULL`.

When I start ipython and look for `itk.L`/`itk.LL` it tells me "module 'itk' has no attribute 'LL'." Specifying that it's signed or unsigned with `S` or `U` seems to fix that problem.